harness/src/harness/service.py

```
@@ -0,0 +1,70 @@
+        return {"status": "ok", "version": "0.0.1"}
+
+    @app.get("/providers")
+    def providers() -> dict:
+        return providers_payload()
+
+    @app.post("/build/stream")
+    async def build_stream(req: BuildRequest) -> EventSourceResponse:
+        _check_schema_version(req.schema_version)
+        agent = agent_factory()
+
+        async def generate():
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `generate`

(ANN202)